src/main/java/frc/robot/commands/ShooterRPM.java

```diff
@@ -29,6 +32,11 @@ public ShooterRPM(Shooter shooter, Vision vision) {
     public void initialize() {
         // System.out.println("STARTING SHOOTER");
         updateSetpoint();
+        if (shooter.getSetpoint() >= (5800 / 60)) {
```

Comment:
I was wrong. Don't do this here. Make a new file and do something similar to this command.